terraform/main.tf

```diff
@@ -0,0 +1,61 @@
+# Enable Google Cloud APIs
+module "enable_google_apis" {
+  source                      = "terraform-google-modules/project-factory/google//modules/project_services"
+  version                     = "~> 13.0"
+
+  project_id                  = var.gcp_project_id
+  disable_services_on_destroy = false
+
+  activate_apis               = var.apis
+}
+
+# Create GKE Cluster
+resource "google_container_cluster" "cluster" {
+  name               = var.name
+  location           = var.region
+  ip_allocation_policy {
+  }
+
+  # Enabling Autopilot for this cluster
+  enable_autopilot = true
```

Comment:
Was it discussed that we make it `autopilot` by default? I think this should be a variable! @NimJay ?

Reply:
Thanks for bring this up, @Shabirmean.
@ahrarmonsur, @jaspermai, and I did talk about this.
I was thinking of using GKE `standard` since its performance and booting is more deterministic.
But there is a push to use `autopilot` as much as possible in our docs (I believe).
Olivier also mentioned that we only run into issues with `autopilot` when using `skaffold`.
We decided use `autopilot` for now and only switch to `standard` if we run into issues. :)